packages/grafana-ui/src/components/Card/Card.tsx

```diff
@@ -243,3 +234,64 @@ export const getCardStyles = stylesFactory((theme: GrafanaTheme, disabled = fals
     `,
   };
 });
+
+interface ChildProps {
+  styles?: ReturnType<typeof getCardStyles>;
+  disabled?: boolean;
+}
+
+const Tags: FC = ({ children }) => {
+  return <>{children}</>;
+};
+Tags.displayName = 'Tags';
+
+const Figure: FC<ChildProps> = ({ children, styles }) => {
+  return <div className={styles?.media}>{children}</div>;
+};
+
+Figure.displayName = 'Figure';
+
+const Meta: FC<ChildProps> = memo(({ children, styles }) => {
+  let meta = children;
+
+  // Join meta data elements by '|'
+  if (Array.isArray(children)) {
+    meta = React.Children.toArray(children).reduce((prev, curr, i) => [
+      prev,
+      <span key={`separator_${i}`} className={styles?.separator}>
+        |
+      </span>,
+      curr,
+    ]);
+  }
+  return <div className={styles?.metadata}>{meta}</div>;
+});
+
+Meta.displayName = 'Meta';
+
+interface ActionsProps extends ChildProps {
+  children: JSX.Element[];
+}
+const Actions: FC<ActionsProps> = ({ children, styles, disabled }) => {
+  return (
+    <div className={styles?.actions}>{React.Children.map(children, child => cloneElement(child, { disabled }))}</div>
+  );
+};
+
+Actions.displayName = 'Actions';
+
+const SecondaryActions: FC<ActionsProps> = ({ children, styles, disabled }) => {
```

Comment:
Seems like the only difference between this and `Actions` is the styles, so probably better idea would be to make `Actions` component accept `variant` prop that would toggle primary or secondary actions styles.

Reply:
:+1: